ext_emconf.php

```diff
@@ -24,7 +24,7 @@
     'depends' => 
     array (
       'php' => '7.0.0-7.1.99',
-      'typo3' => '8.7.0-8.7.99',
+      'typo3' => '8.7.0-9.2.99',
```

Comment:
Please separate this into a second pull request and label it a `[FEATURE]` - then it gets automatically included in the generated changelog and is very visible ;)